.github/workflows/one-click-ga-release.yml

```diff
@@ -0,0 +1,168 @@
+name: One Click GA Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      pre_release:
+        type: boolean
+        description: "Pre-release?"
+        default: true
+        required: false
+      channel:
+        required: true
+        type: choice
+        description: Select a channel
+        options:
+          - ga
+          - beta
+
+env:
+  CHANNEL: ${{inputs.channel}}
+  CHANGELOG_FILE: "temp-changelog.txt"
+
+permissions:
+  contents: write
+  id-token: "write"
+
+jobs:
+  release:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: checkout branch
+        uses: actions/checkout@v3
+        with:
+          ref: ${{inputs.branch}}
```

Comment:
Where the branch value is coming from?
I don't see the `branch`  defined in the inputs section.

Reply:
@davseve its always there, whenever you *manually* run an action
<img width="327" alt="image" src="https://github.com/elementor/elementor/assets/83912471/899b0be8-349e-4f58-a1dd-aa03f52f77b0">